src/Compilers/CSharp/Portable/Symbols/Source/SourceMemberContainerSymbol_ImplementationChecks.cs

```diff
@@ -835,9 +835,10 @@ private void CheckNewModifier(Symbol symbol, bool isNew, DiagnosticBag diagnosti
                             for (int i = 0; i < overridingParameters.Length; i++)
```

Comment:
Nit: this whole loop can just be converted to a foreach, we don't use `i` anywhere except accessing `overridingParameters`.

Reply:
Ah dang, and I thought I had gone through thoroughly here to avoid looking blind.

---
In reply to: [238500830](https://github.com/dotnet/roslyn/pull/31499#discussion_r238500830) [](ancestors = 238500830,238440431)